CONTRIBUTION.md

```diff
@@ -0,0 +1,103 @@
+- What you actually expected to do and why
+    - Ex) `DB4S settings are initalized normally.`
+- System environment, DB4S version, loaded extensions
+    - Ex) `macOS Catalina Version 10.15.6(19G2021), DB4S Version 3.12.0, No Extensions`
+- Attach screenshots or screencasts related to the bug/issue
+    - You can add images can be pasted directly into the GitHub issue,  
+    but screencast can be hosted on streamable.com and the link pasted into the issue.
+- Crash log (If the crash log is known, although this is **NOT** required)
+    - If you have a long crash log, [Pastebin](https://pastebin.com/) can help (Make sure to set expiration to 'never')
+- Project file or database file in use (**NOT** required)
+
+A template from the GitHub issue will help you write a good bug/issue report.
+
+### Suggest enhancement
+Please check the following before suggesting an enhancement.
+
+- Search for existing similar issues
+
+And please follow the below to help members identify the suggestion.
+
+- Use a concise but descriptive title to identify the suggestion
+- Describe the detailed steps for the enhancement you want to propose
+- Attach a simple example of the proposal, already applied projects or screenshots
+
+### Translation
+Project translation can be done through the ['Qt Linguist'](http://doc.qt.io/qt-5/qtlinguist-index.html) program.
+Languages that are already being translated can be translated by modifying the corresponding translation file,
+and translations for new languages can be translated by adding a new entry to the TRANSLATION section of the src/src.pro file and then rebuilding the project.
+
+#### DO NOT translate key names in shortcuts
+Keyboard shortcuts get automatically localized by Qt, and translating key names may indeed break them. They show up on the translation file only for the rare occasions that one has to redirect a shortcut to another one in a specific language. But even then, you should use the English key names to get this working. Theoretically, one could follow Qt conventions for translated key names, and they should work, but the risk to stumble upon Qt bugs or moving conventions is high, so it is better to not translate keys and only remap the shortcut. If you want to preserve the original key sequence (recommended), just leave the translation empty and finished.
+
+## Styleguide
+Styleguides help keep the project's source code looking good and consistent.
+This can be of great help to existing and new contributors.
+
+### CPP source code
+#### Indent
+- 4 spaces(**NOT** tabs) for indentation
+
+#### Declare variables
+- Use lowerCamelCase
+- Declare only one variable per line
+- \* in pointer variable is attached to data type
+    - Ex) `int* ptr`
+- Avoid meaningless names (although these can be used for counter or temporary names)
+
+#### Whitespace
+- Do **NOT** use spaces after keywords
+- Leave a space after each comma
+
+#### Braces
+- Place braces on the line after the keyword
```

Comment:
It is unclear if this includes the body of functions. Currently, most of them place the opening brace in the line following the prototype.

Reply:
I modified it as you said.
> Place braces on the same line as the keyword

Reply:
I didn't exactly mean that. My impression is that the brace in the following line is more prevalent for functions, and in the same line, more prevalent for control statements, but it's true that currently there isn't any consensus.